mappings/net/minecraft/client/render/entity/model/BookModel.mapping

```diff
@@ -8,3 +8,12 @@ CLASS net/minecraft/class_557 net/minecraft/client/render/entity/model/BookModel
 	FIELD field_3338 rightCover Lnet/minecraft/class_630;
 	FIELD field_3339 rightPage Lnet/minecraft/class_630;
 	METHOD method_17073 setPageAngles (FFFF)V
+	METHOD method_24184 (Lnet/minecraft/class_4587;Lnet/minecraft/class_4588;IIFFFF)V
+		ARG 1 matrices
+		ARG 2 vertices
```

Comment:
`vertexConsumer`.

Reply:
If it's a VertexConsumer naming it `vertices` is intended.